### PR TITLE
Fixing gulpfile so that live reload works on the theme stylesheets

### PR DESCRIPTION
**Pull request by etsauer**
#### What is this PR About?
Addresses issue #61 

#### How should we test or review this PR?

Start live preview with `npm start`. Then make a change to the theme scss, save it, and make sure you can see the change when the browser reloads.

#### Is there a relevant Trello card or Github issue open for this?
#61 

#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this

-----

- [x] Have you followed the [contributing guidelines](https://github.com/redhat-cop/uncontained.io/blob/master/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the uncontained.io guides?

-----

**Please note: we may close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by rdebeasi** [comment]
Cool, thank you! I actually just ran into this while reviewing another issue.

I'm having trouble getting this fix working reliably. If I make a sass change, the page reloads but I don't see my update. If I stop npm, start it again, and save the file again, _then_ I see my change.

If the npm build is getting to fiddly to be worthwhile, another option would be to use [Hugo pieplines](https://gohugo.io/news/0.43-relnotes/), which were added literally two weeks after you drafted the PR to set up Victor Hugo. Open source, amirite? :upside_down_face:

**Comment by etsauer** [comment]
@rdebeasi strange. I've been using this fix pretty consistently for the past week and I've noticed no issues. What could be different about our environments?

**Comment by rdebeasi** [comment]
Weird! I wonder if we're using it differently. For me, sass files are only built when gulp starts, not when the files are changed.

I tested by running these commands to watch CSS builds ([via Stack Overflow](https://superuser.com/a/1056022/284351):

```
cd site/themes/uncontained.io/static/dist/css
watch -d -t -g ls -lR && echo "changed"
```

To trigger sass builds on each save, I tried adding this line to `runServer`:
```
  gulp.watch("./site/themes/uncontained.io/src/scss/**/*.scss", ["sass"]);
```

That resulted in the sass being built with each save, but hugo builds seemed to complete before the sass file build completed. The preview reloaded each time I changed the sass file, but it was always one update "behind" the actual update. That sent me down the rabbit hole of trying to set up sequential tasks, which requires either a plugin or Gulp 4.

Anyway, this PR is an improvement overall, so I'm happy to approve it and solve the other issues later.

File: gulpfile.babel.js [hunk 1]
```diff
@@ -88,10 +88,8 @@ function runServer() {
       baseDir: "./dist"
     }
   });
-  gulp.watch("./src/js/**/*.js", ["js"]);
-  gulp.watch("./src/css/**/*.css", ["css"]);
-  gulp.watch("./src/fonts/**/*", ["fonts"]);
-  gulp.watch("./site/**/*", ["hugo"]);
+  gulp.watch("./site/themes/*/src/**/*.scss", ["sass"])
+  gulp.watch(["./site/**/*", "!./site/themes/*/src/**"], ["hugo"]);
```
Comment (jimdillon) [inline gulpfile.babel.js]: Mighty fine troubleshooting to fix this one!

**Pull request merged**